docs/legacy/common-problems.asciidoc

```diff
@@ -325,9 +298,10 @@ APM agent --> Load Balancer  --> APM Server
 
 *If Elasticsearch is down*
 
-If Elasticsearch goes down, APM will keep some data in its internal in-memory queue.
-When the queue fills up, APM will respond with <<queue-full-legacy,`HTTP 503: Queue is full`>> and
-block requests until there is space available or until the request timeout is reached.
+APM Server does not have an internal queue to buffer requests,
+but instead leverages an HTTP request timeout to act as back-pressure.
+If Elasticsearch goes down, the APM Server will eventually deny incoming requests.
+Both the APM Server and APM agent(s) will issue logs accordingly.
```

Comment:
👋 @marclop Is this accurate? Is there a better way to describe the new model indexer behavior? (I took some of your words from https://github.com/elastic/sdh-apm/issues/542#issuecomment-1077818731)

I also used this wording to replace other "queue is full" docs—please see https://github.com/elastic/apm-server/pull/7901/commits/973a63e45736ee16f09b388e888e83de189f4bc0.